catalog/core/catalog-core-contentresourcereader/src/test/java/org/codice/ddf/catalog/content/resource/reader/DerivedContentActionProviderTest.java

```diff
@@ -64,7 +64,7 @@
 
     @BeforeClass
     public static void init() throws URISyntaxException {
-        actionUri = new URI("https://example.com/download?transform=resource");
+        actionUri = new URI("https://example.com/download?transform=resource&qualifier="+QUALIFIER_VALUE);
```

Comment:
Looks like a formatting issue. There should be a space before and after the plus-sign.